framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -135,8 +135,8 @@ int cfg_get_rdkb_extensions()
 {
     int retVal = -1;
     if (cfg_get_prplmesh_param_int("rdkb_extensions", &retVal) == RETURN_ERR) {
-        printf("cfg_get_rdkb_extensions: Failed to read RDKB Extensions parameter\n");
-        return RETURN_ERR;
+        MAPF_INFO("cfg_get_rdkb_extensions: Failed to read RDKB Extensions parameter\n");
```

Comment:
```suggestion
        MAPF_INFO("cfg_get_rdkb_extensions: Failed to read RDKB Extensions parameter, using default " << int(DEFAULT_RDKB_EXTENSIONS));
```

Reply:
Same for all others - it is useful to see in the logs what is the default in case it will change in the future and we are debugging an old prplmesh.